bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/SingleProbeStrategy.scala

```diff
@@ -250,8 +251,8 @@ object SingleProbeStrategy {
     val ra1    = st.coordinates.ra.toAngle.toRadians
     val dec1   = st.coordinates.dec.toAngle.toRadians
     val target = HmsDegTarget.fromSkyObject(st.toOldModel)
-    val ra2    = Angle.fromDegrees(target.getRaDegrees).toRadians
-    val dec2   = Angle.fromDegrees(target.getDecDegrees).toRadians
+    val ra2    = Angle.fromDegrees(target.getRa.getAs(Units.DEGREES)).toRadians
+    val dec2   = Angle.fromDegrees(target.getDec.getAs(Units.DEGREES)).toRadians
     val raDiff = ra2 - ra1
```

Comment:
Here we know we have a sidereal target so we can get the coordinates directly. Once we start correcting for proper motion this will need a time as well.